hack/prowimagebuilder/main.go

```diff
@@ -217,23 +217,26 @@ func buildAndPush(id *imageDef, koDockerRepo string, push bool) error {
 	if err := setup(id); err != nil {
 		return fmt.Errorf("setup: %w", err)
 	}
-	_, err = runCmd("_bin/ko", publishArgs...)
-	return err
+	for _, dockerRepo := range dockerRepos {
```

Comment:
maybe a comment mentioning that ko only supports on repo at a time?

alternatively does it make sense to copy over the images between repos instead of rebuilding? (or perhaps a future TODO)

Reply:
fat finger, accidentally resolved

Reply:
good point, added a comment.

ko has built in cache, which makes the subsequent `ko` calls relatively cheap, so we should be fine here